src/algebra/algebra/subalgebra.lean

```diff
@@ -540,6 +540,11 @@ lemma mul_mem_sup {S T : subalgebra R A} {x y : A} (hx : x ∈ S) (hy : y ∈ T)
   x * y ∈ S ⊔ T :=
 (S ⊔ T).mul_mem (mem_sup_left hx) (mem_sup_right hy)
 
+lemma map_sup (f : A →ₐ[R] B) (S T : subalgebra R A) : (S ⊔ T).map f = S.map f ⊔ T.map f :=
+le_antisymm (subalgebra.map_le.mpr
+  (sup_le (subalgebra.map_le.mp le_sup_left) (subalgebra.map_le.mp le_sup_right)))
+  (sup_le (subalgebra.map_mono le_sup_left) (subalgebra.map_mono le_sup_right))
```

Comment:
Do you need the `subalgebra.` prefices here?

Reply:
Seems so.